src/routes/landing/nearbyPlaces.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import { CityModel } from '@integreat-app/integreat-api-client'
+
+const NUMBER_OF_CLOSEST_CITIES = 3
+const MAXIMAL_DISTANCE = 90
+
+const degreesToRadians = (deg: number): number => {
+  const degreesSemicircle = 180
+  return Math.PI * deg / degreesSemicircle
+}
+
+const calculateDistance = (longitude0: number, latitude0: number, longitude1: number, latitude1: number): number => {
+  const earthRadius = 6371
```

Comment:
Should probably be a contant and all caps magic value. Like

const EARTH_RADIUS = 314

Reply:
I just moved the code and didn't modify it.